Comanche055/P76.agc

```diff
@@ -35,10 +35,10 @@
 #	TIME (TIG) AT WHICH THE OTHER VEHICLE VELOCITY WAS CHANGED BY EXECUTION OF A THRUSTING MANEUVER. THE
 #	OTHER VEHICLE STATE VECTOR IS INTEGRATED TO TIG AND UPDATED BY THE ADDITION OF DELTA V (DELTA V HAVING
 #	BEEN TRANSFORMED FROM LV TO REF COSYS).  USING INTEGRVS, THE PROGRAM THEN INTEGRATES THE OTHER
-#	VEHICLE STATE VECTOR TO THE STATE VECTOR OF THIS VEHICLE, THUS INSURING THAT THE W-MATRIX AND BOTH VEHICLE
+#	VEHICLE STATE VECTOR TO THE STATE VECTOR OF THIS VEHICLE. THUS INSURING THAT THE W-MATRIX AND BOTH VEHICLE
```

Comment:
❌ This is yet another badly printed comma - you can partially see the flick through the jpeg artefacts